Builder/BuilderInterface.php

```diff
@@ -22,8 +22,11 @@
 interface BuilderInterface
 {
     /**
-     * @param AdminInterface            $admin
-     * @param FieldDescriptionInterface $fieldDescription
+     * Adds missing information to the given field description from the model manager metadata,
```

Comment:
> what a "real" builder is

The `DatagridBuilder`, `ListBuilder` & `ShowBuilder` concrete implementations, i.e `SonataDoctrineORMAdminBundle\Builder\ListBuilder` et al.